### ISSUE TITLE
Fonts not installing

### ISSUE THREAD

**Issue by erossound** [photopea/photopea#3669]
Fonts are not showing up after loading in. I've tried different ways and different file types including otf and ttf but fonts are not showing up when I attempt to load them in. A notification appears that the font was loaded but it doesn't show in the font library. I tried the file open method and also the load font button within the font browser but no luck.

**Comment by erossound** [comment]
Would also like to know if it's possible to upload multiple fonts at once or if it must be done 1 by 1. I've tried both but neither worked.

**Comment by photopea** [comment]
The fonts should be added to the bottom of the font list. Could you share your font with us?

Sure, you should be able to load as many fonts as you want at the same time.

**Comment by grubnight** [comment]
fonts library not loading?  struck on one font option

**Comment by JuanmaAranda** [comment]
The same thing happens to me. The fonts, despite appearing in Edit > Local Storage, do not appear later in the font listing. 
Neither at the end, nor in alphabetical order, nor anywhere.
Tested from different computers, both with Windows and iOs.

**Comment by photopea** [comment]
After you load a TTF or OTF file into Photopea, it should appear in the font list. Please, show us a font, which does not work in Photopea, so that we can fix the bug.

**Comment by JuanmaAranda** [comment]
Example: Fortnite.ttf

**Comment by photopea** [comment]
@JuanmaAranda Could you share that Fortnite.ttf with us?

**Comment by JuanmaAranda** [comment]
[fortnite.zip](https://github.com/photopea/photopea/files/7072934/fortnite.zip)

**Comment by photopea** [comment]
The font which you provided is called "Burbank Big Condensed". When you rename the file to "fortnite" or any other word, it does not change the name of a font inside the file. The font creator set the font name, family name, and other parameters.

I loaded your TTF file to Photopea and it worked correctly, I was able to use it.
![image](https://user-images.githubusercontent.com/15955558/131261146-e784d7ea-90fc-4317-975e-6bb9366e32d7.png)
